native/src/routes/LandingContainer.tsx

```diff
@@ -67,20 +67,11 @@ const mapStateToProps = (state: StateType, ownProps: OwnProps): StateProps => {
 }
 
 const LandingContainer = ({ navigation, dispatch, cities, language, route: _route }: LandingContainerProps) => {
-  const navigateToDashboard = (cityCode: string, languageCode: string) => {
-    navigateToCategory({
-      routeName: DASHBOARD_ROUTE,
-      navigation,
-      dispatch,
-      cityCode,
-      languageCode,
-      cityContentPath: cityContentPath({
-        cityCode,
-        languageCode,
-      }),
-      forceRefresh: false,
-      resetNavigation: true,
-    })
+  const { changeCityCode } = useContext(AppContext)
+
+  const navigateToDashboard = (cityCode: string) => {
+    changeCityCode(cityCode)
+    navigation.reset({ index: 0, routes: [{ name: CATEGORIES_ROUTE, params: {} }] })
```

Comment:
In general, does this also work if you deep link into the app from cold start?

Reply:
Probably not yet but its on my list and there is a TODO in `navigateToDeepLink`

Reply:
Added a todo here: https://github.com/digitalfabrik/integreat-app/pull/802